src/components/lyne-cta-button/lyne-cta-button.scss

```diff
@@ -121,44 +136,97 @@
   --color-hover-text: var(--color-iron-default);
 }
 
+.button--small {
+  --border-width: 1;
+  --font-size: 12;
+  --min-height: 40;
+
+  --padding-block: var(--spacing-fixed-2x);
+  --padding-inline: var(--spacing-fixed-5x)
+}
+
+.button--icon-only {
+  --padding-inline: 0;
+  --icon-margin-inline-end: 0;
+
+  width: pxToRem(var(--min-height));
+  justify-content: center;
+}
+
 .button {
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  box-sizing: border-box;
   display: flex;
   align-items: center;
-  padding: 0 pxToRem(var(--spacing-fixed-10x));
-  height: pxToRem(var(--spacing-fixed-10x));
-  transition: background-color cubic-bezier(.47, .1, 1, .63) 200ms;
-  cursor: pointer;
-  border-radius: 0.1rem;
+  transition-duration: var(--transition-duration);
+  transition-timing-function: var(--transition-easing-function);
+  transition-property: background-color, color;
+  min-height: pxToRem(var(--min-height));
+  margin: 0;
   outline: none;
-  -webkit-appearance: none;
-  -moz-appearance: none;
-  color: var(--color-default-text);
+  border: pxToRem(var(--border-width)) solid var(--color-default-border);
+  border-radius: pxToRem(var(--border-radius));
   background-color: var(--color-default-background);
-  border: pxToRem(1.5) solid var(--color-default-border);
+  cursor: pointer;
```

Comment:
I know that we did it before and other design systems do it as well. There was quite some conversation at some point whether this is the right way to do: https://medium.com/simple-human/buttons-shouldnt-have-a-hand-cursor-b11e99ca374b

Reply:
Was not aware of this problematic, so thanks for the eduaction 🤗